src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const body = document.querySelector('tbody');
+
+function sortTab(index) {
+  const rows = Array.from(body.querySelectorAll('tr'));
+
+  rows.sort((rowA, rowB) => {
+    const cellA = rowA.cells[index].textContent.trim();
+    const cellB = rowB.cells[index].textContent.trim();
+
+    return cellA.localeCompare(cellB, undefined, { numeric: true });
```

Comment:
Using `localeCompare` with the `numeric: true` option is a good choice for sorting numbers as strings. Ensure that all data in the column is consistently formatted as numbers or strings to avoid unexpected behavior.